parm/atm/obs/config/sondes.yaml

```diff
@@ -184,150 +91,14 @@ obs filters:
         options:
           error_min: 100         # 1 mb
```

Comment:
Based on our Wednesday group discussion, I think we need to make clear these are 'floats' and not integers (I think there are a few spots throughout this YAML).

Reply:
@CoryMartin-NOAA fixed.